### DISCUSSION TITLE
Installation without Docker?

### DISCUSSION THREAD

**Discussion by neidiom** [postalserver/postal#2871] [Help with using Postal]
Is it possible?

**Comment by willpower232** [comment]
Technically yes but you would have to manage each of the required processes yourself with something like supervisord or even systemd I guess. It would definitely be much easier to use docker and docker compose.

**Comment by adamcooke** [comment]
Yes, running Postal without Docker is do-able. This is the recommended approach for anyone who wants to actually do any development work on Postal - I do it myself. However, to run in production you would also need to add additional components (as Will says) to actually manage the running processes.

There are some docs about how to set this up: https://github.com/postalserver/postal/blob/main/CONTRIBUTING.md
